packages/swap/src/utils/getPoolQuote.ts

```diff
@@ -110,6 +110,7 @@ export default async function getPoolQuote({
     egressAmount: outputAmount.toString(),
     includedFees,
     lowLiquidityWarning,
+    inputUsdValue: boostFeeBps ? inputUsdValue : undefined,
```

Comment:
why are we checking for `boostFeeBps` here?

Reply:
oh it's the wrong way around. i don't want to include it on boosted quotes